test/test_momentum_op.cpp

```diff
@@ -96,7 +97,9 @@ TEST(ReplaceModifier, Momentum1) {
   std::vector<tensorflow::Tensor> tf_outputs1;
   std::vector<tensorflow::Tensor> tf_outputs2;
   std::vector<tensorflow::Tensor> tf_outputs3;
+
   ASSERT_OK(tf_session.Run({{var_assign, accum_assign}}, &tf_outputs1));
+  std::vector<string> tf_outputs1_s = ConvertToString(tf_outputs1, 10);
```

Comment:
change the 10 that is hard coded to something that is computed.

Reply:
done